### PR TITLE
ENT-7465: Insured exported reports from Mission Portal are in the correct location

### PR DESCRIPTION
**Pull request by nickanderson**
Ticket: ENT-7465
Changelog: Title

### CODE REVIEW

File: cfe_internal/enterprise/CFE_hub_specific.cf [hunk 1]
```diff
@@ -570,6 +570,27 @@ bundle agent cfe_internal_enterprise_maintenance
...
+{
+  meta:
+    am_policy_hub.enterprise_edition::
+      "tags" slist => { "enterprise_maintenance" };
+
+  files:
+
+    am_policy_hub.enterprise_edition::
+
+      "$(cfe_internal_hub_vars.public_docroot)/tmp/." -> { "ENT-7465" }
```
Comment (aleksandrychev) [inline cfe_internal/enterprise/CFE_hub_specific.cf:584]: in 3.18 (htdocs/public/tmp) and 3.15 (htdocs/tmp) tmp has different location, I'd say to add variable and use it as bundle:
 ```
 vars:
    "tmp_directories_list" slist => {"$(cfe_internal_hub_vars.public_docroot)/tmp/.", "$(cfe_internal_hub_vars.docroot)/tmp/."};
```

File: cfe_internal/enterprise/CFE_hub_specific.cf [hunk 2]
```diff
@@ -570,6 +570,27 @@ bundle agent cfe_internal_enterprise_maintenance
...
+        depth_search => recurse( inf ),
+        file_select => by_name( '.*\.(csv|pdf)' ),
+        transformer => "/bin/mv $(this.promiser) $(cfe_internal_hub_vars.public_docroot)/static/",
```
Comment (aleksandrychev) [inline cfe_internal/enterprise/CFE_hub_specific.cf:587]: `$(cfe_internal_hub_vars.public_docroot)/static/`
should be 
`$(cfe_internal_hub_vars.docroot)/static/`

File: cfe_internal/enterprise/CFE_hub_specific.cf [hunk 3]
```diff
@@ -570,6 +570,27 @@ bundle agent cfe_internal_enterprise_maintenance
...
+        if => isdir( "$(cfe_internal_hub_vars.public_docroot)/static/." ),
```
Comment (aleksandrychev) [inline cfe_internal/enterprise/CFE_hub_specific.cf:588]: `$(cfe_internal_hub_vars.public_docroot)/static/`
should be 
`$(cfe_internal_hub_vars.docroot)/static/`

**Review by aleksandrychev — CHANGES_REQUESTED** [review]

**Comment by nickanderson** [comment]
Superseded by https://github.com/cfengine/masterfiles/pull/2067

**Pull request closed without merging**